gdpr/gdpr/AppDelegate.m

```diff
@@ -0,0 +1,71 @@
+//
+//  AppDelegate.m
+//  gdpr
+//
+//  Created by Kaleb Bataran on 4/3/18.
+//  Copyright © 2018 localytics. All rights reserved.
+//
+
+#import "AppDelegate.h"
+@import Localytics;
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    // Override point for customization after application launch.
+    
+    [Localytics pauseDataUploading:YES];
+    [Localytics autoIntegrate:@"a39f6f0cb378dbd8239767d-487f3e04-8691-11e6-ab4e-0098ae15fcaa"
+        withLocalyticsOptions:@{
+                                LOCALYTICS_WIFI_UPLOAD_INTERVAL_SECONDS: @5,
+                                LOCALYTICS_GREAT_NETWORK_UPLOAD_INTERVAL_SECONDS: @10,
+                                LOCALYTICS_DECENT_NETWORK_UPLOAD_INTERVAL_SECONDS: @30,
+                                LOCALYTICS_BAD_NETWORK_UPLOAD_INTERVAL_SECONDS: @90
+                                }
+                launchOptions:launchOptions];
+    
+    if ([[NSUserDefaults standardUserDefaults] boolForKey:@"isLoggedIn"]) {
+        //TODO check user's opt-out status on your server
+        BOOL optOutStatus = NO;
+        [Localytics setPrivacyOptedOut:optOutStatus];
+        //go to home screen
+        self.window.rootViewController = [self.window.rootViewController.storyboard instantiateViewControllerWithIdentifier:@"HomeVC"];
```

Comment:
Customers would want their app to be useable even when the user has not opted out. So the rootViewController should be rendered and the flow of login shown. cannot do authentication without an UI.

Reply:
not sure i understand your comment, why would I show the login viewcontroller if the user is already logged in?

Reply:
self.window.rootViewController = [self.window.rootViewController.storyboard instantiateViewControllerWithIdentifier:@"HomeVC"];
 Is within an if what happens in the else case?

Reply:
Not sure what you mean by this ^ Can you explain?